### DISCUSSION TITLE
Can we support navigate to the method in base or derived class?

### DISCUSSION THREAD

**Discussion by shenmishajing** [microsoft/pylance-release#3772] [Enhancement]
In PyCharm, we can navigate to the same method in base or derived class by just one click. Can we do this in vscode or will we support it?

For example, think about code like follow:
```python
class A
    def print_hello():
        print('I am in A')

class B
    def print_hello():
        print('I am in B')

class C
    def print_hello():
        print('I am in C')
```

When we write this in PyCharm, we will get a up arrow on the left of the print_hello method of B and C, and if we click this, we will navigate to the print_hello method of A. Similerly, we will get a down arrow on the left of the print_hello method of A, and if we click this, we can select B or C to navigate to. It will be helpful if vscode support this also.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
